compiler/tests/generated/apply/basic_types_apply.h

```diff
@@ -6,208 +6,257 @@
 #include <bond/stream/output_buffer.h>
 
 
-namespace tests
+namespace bond
```

Comment:
Since we are moving namespaces anyway, would it makes sense to move these inside bond::details?